### PR TITLE
feat(Divider): render content inside

### PR DESCRIPTION
**Pull request by blasdfaa**

### CODE REVIEW

**Comment by blasdfaa** [comment]
@bamje can i request more reviewers?

**Comment by v4dyar4** [comment]
Hi! Thx for the PR. I will do a review of PR very soon
@blasdfaa

File: src/components/Divider/__tests__/Divider.test.tsx [hunk 1]
```diff
@@ -36,4 +37,25 @@ describe('Divider', () => {
...
+        );
+        const content = screen.getByText(childrenText);
+
+        expect(content).toBeVisible();
+    });
+    test.each(new Array<DividerAlign>('start', 'center', 'end'))(
+        'Should render children content with given align "%s"',
+        (align) => {
+            render(<Divider align={align} qa={qa} />);
+            const button = screen.getByTestId(qa);
```
Comment (v4dyar4) [inline src/components/Divider/__tests__/Divider.test.tsx:56]: Why is this a ```button```? Maybe rename it to ```element``` or something else?

**Review by v4dyar4 — COMMENTED** [review]
Also, please complete README.MD for GITHUB_BLOCK sections

**Comment by blasdfaa** [comment]
@v4dyar4 ready

File: src/components/Divider/README.md [hunk 2]
```diff
@@ -132,12 +132,101 @@ import {Flex, Label, Divider} from '@gravity-ui/uikit';
...
+
 ### Properties
 
 | Name        | Description                             | Type                    | Default      |
 | :---------- | :-------------------------------------- | :---------------------- | :----------- |
 | className   | HTML `class` attribute                  | `string`                | -            |
 | orientation | Sets the direction of divider           | `horizontal - vertical` | `horizontal` |
+| align       | Custom content position                 | `start - center - end`  | `start`      |
 | style       | HTML `style` attribute                  | `React.CSSProperties`   |              |
 | qa          | HTML `data-qa` attribute, used in tests | `string`                |              |
```
Comment (v4dyar4) [inline src/components/Divider/README.md:232]: please, add `children?:ReactNode`

**Comment by blasdfaa** [comment]
@v4dyar4 added

**Review by v4dyar4 — APPROVED** [review]

**Comment by v4dyar4** [comment]
@blasdfaa Could you rebase branch and merge it?

**Comment by blasdfaa** [comment]
> @blasdfaa Could you rebase branch and merge it?

approve please

**Review by v4dyar4 — APPROVED** [review]

**Comment by blasdfaa** [comment]
@v4dyar4 I don't have the necessary access to merge

**Pull request merged**